server/lib/resolvers/resolverMapper.js

```diff
@@ -50,6 +51,8 @@ module.exports = function (dataSources, resolverMappings, pubsub) {
       compiledResponseMapping
     )
 
+    resolver = wrapResolverWithHooks(resolver, resolverMapping)
```

Comment:
IMO this should be something like

```js
if (resolverMapping.preHook || resolverMapping.postHook) {
  resolver = wrapResolverWithHooks(resolver, resolverMapping)
}
```

Otherwise we are needlessly adding an additional layer of function calls and try catch blocks for every request. I know it's a minor thing but it can be easily avoided.

You can still perform the `isEmpty` checks within the `wrapResolverWithHooks` function.